src/main/java/com/javagirls/MangoRoom/repository/ReservationRepository.java

```diff
@@ -1,10 +1,15 @@
 package com.javagirls.MangoRoom.repository;
 
+import java.util.List;
+
 import com.javagirls.MangoRoom.entity.Reservation;
+import com.javagirls.MangoRoom.entity.Room;
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.stereotype.Repository;
 
 @Repository
 public interface ReservationRepository extends JpaRepository<Reservation, Long> {
 
+	List<Reservation> findByRoom(Room room);
```

Comment:
nie wydaje mi się aby to mogło zadziałać, a nawet jeśli to bardziej optymalne będzie szukanie po id

Reply:
w RoomService wyszukuję pokój po id i dopiero wtedy go przekazuję do tej metody.